core/vm/eips.go

```diff
@@ -46,9 +45,9 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
 // - Define SELFBALANCE, with cost GasFastStep (5)
 func enable1884(jt *JumpTable) {
 	// Gas cost changes
-	jt[BALANCE].constantGas = params.BalanceGasEIP1884
-	jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
-	jt[SLOAD].constantGas = params.SloadGasEIP1884
+	// jt[BALANCE].constantGas = params.BalanceGasEIP1884
+	// jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
+	// jt[SLOAD].constantGas = params.SloadGasEIP1884
```

Comment:
Adding a comment for any confused future readers.
```suggestion
	
	// Celo does not include this gas change at genesis, where the rest of EIP1884 is applied.
	// It is planned to be applied at a later date.
	// jt[SLOAD].constantGas = params.SloadGasEIP1884
```